### DISCUSSION TITLE
Option to resume a Youtube video at the time before the site was blocked

### DISCUSSION THREAD

**Discussion by dawin64** [proginosko/LeechBlockNG#425] [Ideas]
I would like to have an option to resume a YouTube video at the point where it was cut off, because if you're watching a long video and it cuts off, it's difficult to find the exact point where it was cut off. I believe you could use ```Math.trunc(document.querySelector('.video-stream').currentTime)``` this will get the current time of the current YouTube video, and set/or append the ```&t=xx``` parameter to the URL before the site gets blocked. (should only apply if you are on https://www.youtube.com/watch?*)

**Comment by poyribn** [comment]
Continuously updating the Timestamp in every YouTube Tab is a complex and specific Task by itself.
So this should not be a Feature in LeechBlock, there should be a Browser Extension that does only that.

**Comment by proginosko** [comment]
A PR was submitted that adds that feature: https://github.com/proginosko/LeechBlockNG/pull/438

However, I haven't merged it into the main branch because (1) I don't want to add scripting permissions to LB and (2) I'm reluctant to start adding website-specific features.

I believe there are other extensions that provide a YouTube resume feature.
